src/views/auth/LoginPage.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <Layout header-type="basic" :is-bottom-nav="false" :show-left-icon="false">
+    <template #content>
+      <div class="login-page">
+        <!-- 1) 로고 + 설명 -->
+        <div class="logo-box">
+          <!-- src/assets/images/logo.png 에 로고 파일 넣어두세요 -->
+          <img src="@/assets/images/danji-logo-main.png" alt="단지 로고" class="logo" />
+          <p class="tagline">
+            단 하나의 지갑이면 충분해요<br />
+            모든 지갑을 하나로, 단지에서 시작하세요 😊
+          </p>
+        </div>
+
+        <!-- 2) 흰 카드 : 입력창 2개 + 링크 -->
+        <div class="form-card">
+          <!-- 감싸는 div 하나 추가 -->
+          <div class="input-wrapper">
+            <danji-input v-model="username" placeholder="아이디를 입력해주세요.">
+              <template #icon> <User class="fake-input-icon" /> </template>
+            </danji-input>
+          </div>
+
+          <div class="input-wrapper">
+            <danji-input v-model="password" type="password" placeholder="비밀번호를 입력해주세요.">
+              <template #icon>
+                <Lock class="fake-input-icon" />
+              </template>
+            </danji-input>
+          </div>
+
+          <router-link to="/signup" class="signup-link">
+            단지의 첫 지갑을 만들어볼까요? <strong>회원가입</strong>
+            <img src="@/assets/icons/right-arrow.svg" alt="화살표" class="arrow-icon" />
```

Comment:
아이콘 넣으실 때 Lucide 사용해주시면 됩니다!
script 섹션에 import 아래처럼 해주시고,
```typescript
import { ChevronRight } from 'lucide-vue-next';
``` 
아래처럼 수정해주시면 됩니다!
```suggestion
            <chevron-right class="text-Gray-3  />
```